integration_tests/dataproc_test_case.py

```diff
@@ -199,6 +199,11 @@ def tearDown(self):
     def getClusterName(self):
         return self.name
 
+    @staticmethod
+    def getBaseOS():
+        # Get the base operating system: '1.5-debian10' -> 'debian' 
+        return re.search("[a-z]+", FLAGS.image_version).group()
```

Comment:
What it will return for `preview-debian10` version?

Reply:
Oops, fixed...